skins/oasis/modules/LatestActivityController.class.php

```diff
@@ -58,6 +59,15 @@ public function executeIndex() {
 		$this->setVal( 'activityIcon', DesignSystemHelper::renderSvg( 'wds-icons-activity-small', 'wds-icon wds-icon-small wds-activity-icon' ) );
 		$this->setVal( 'renderCommunityEntryPoint', !empty( $wgEnableCommunityPageExt ) );
 
+		// TODO https://wikia-inc.atlassian.net/browse/CAKE-4746
+		if ( $wgEnableTriviaQuizzesExt && in_array($wgTitle->getText(), $wgTriviaQuizzesEnabledPages) ) {
+            $this->setVal( 'renderTriviaQuizzes', $wgEnableTriviaQuizzesExt );
+            // TODO use this string instead when we have i18n support, for now, hard code the string for all languages
+            // $this->setVal( 'moduleHeader', wfMessage('trivia-quizzes-featured-quizzes-header')->escaped() );
```

Comment:
Would it be worth just using this now? All other languages will fallback to the English string if no translations exist anyway, so right now it would be the same result as hardcoding and avoids needing to change it later, since once translations are added, they'll just work.

Reply:
Good to know, thanks!

Reply:
Fixed in 93c4c1a7be7